### ISSUE TITLE
Test hangs up till the browser connection termination if the tested server is not running

### ISSUE THREAD

**Issue by inikulin** [DevExpress/testcafe#367]
Just run in TestCafe dir:
```
node bin/testcafe chrome test/functional/fixtures/quarantine-mode/index.test.js
```

**Comment by AndreyBelym** [comment]
Caused by DevExpress/testcafe-hammerhead#446.

**Comment by AndreyBelym** [comment]
DevExpress/testcafe-hammerhead#447 fixed it for me. @inikulin, please check it on your side.

**Comment by inikulin** [comment]
@AndreyBelym we need to update hammerhead first and then close the issue.

**Comment by AndreyBelym** [comment]
Fixed since `testcafe-hammerhead@6.0.1`.
